src/shared/script/shared-code/langUtils.ts

```diff
@@ -1,6 +1,21 @@
 import { PLNumber, PLFractionNumber, plString, PLString } from 'pocket-lisp-stdlib'
 import { assertInteger } from './utils'
 
+const translations = {
+  hét: 'wk',
+  ó: 'h',
+  óra: 'h',
+  p: 'min',
+  perc: 'min',
+  mp: 's',
+  tonna: 't',
+  hüvelyk: 'in',
+}
+
+export function translate(str: string): string {
+  return str in translations ? translations[str] : str
```

Comment:
In general I do not recommend to use `in` operator in JavaScript
`in` check if the key is exists in the object or not but it will return with true even if the value of the key is undefined:
```
"a" in {} // => false
"a" in { a: undefined } // => true 
```
I recommend to do this instead:
```suggestion
  return translations[str] || str
```